test/functional/users_controller_test.rb

```diff
@@ -210,7 +210,7 @@ class UsersControllerTest < ActionController::TestCase
     @controller.stubs(:available_sso).returns(@sso)
     @controller.stubs(:get_sso_method).returns(@sso)
     get :extlogin, {}, {}
-    get :logout, {}, {}
+    post :logout, {}, {}
```

Comment:
> What was the accident?

GET for :logout didn't exist until now. Functional tests don't go through routes so even PUT :logout would work. But now when GET got a different logic it stopped redirecting the user.

> How do we prevent from happening again?

not sure, how about stop writing functional tests? :-)